tozny/resource_realm_group.go

```diff
@@ -76,10 +77,132 @@ func resourceRealmGroup() *schema.Resource {
 				Computed:    true,
 				ForceNew:    true,
 			},
+			"access_policies": {
+				Description: "The entities which control granting temporary access to groups through multi-party control.",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"access_policy_id": {
+							Description: "Server generated Id for the access policy",
+							Type:        schema.TypeInt,
+							Computed:    true,
+						},
+						"approval_role_ids": {
+							Description: "The role that can approve requests for groups with this access policy.",
+							Type:        schema.TypeList,
+							MinItems:    1,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+							Required: true,
+						},
+						"required_approvals": {
+							Description: "The number of approvals required for multi-party control of this group.",
```

Comment:
Does the server set the deault to 1 or should we set it here?

Reply:
Server does set a default to 1

Reply:
lets make that explicit here with a "Default 1"

Reply:
Good idea 👍